ts/tsconfig.json

```diff
@@ -17,7 +17,7 @@
         // "lib": [],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
         "lib": [
             "DOM",
-            "ES2016"
+            "ES2017"
```

Comment:
We need IE11 polyfill for `Object.entries` and `Object.values`, right?

Also it is better to keep `ES2016` and add `ES2017.Object`, to keep the amount of required polyfills in check.

Reply:
I'll redo this, I used `Object.values` temporarily, but undid it.

Reply:
Looks like the polyfills are already there, so I'll just add `ES2027.Object`.

https://github.com/highcharts/highcharts/blob/234497b961fede9457e1943e0c4529fe144aefc7/ts/masters-es5/highcharts.src.ts#L48-L73